### ISSUE TITLE
Training Time on two RTX 4090 GPUs

### ISSUE THREAD

**Issue by hoo223** [AsukaCamellia/TCPFormer#2]
Thank you for sharing your great work.

In the paper, you mentioned that training was conducted using two RTX 4090 GPUs and batch size of 16. Could you share how long the training process took for both the 81-frame and 243-frame models? 

And, in the config files, I found that the batch size setting differs from 16 (4 for 243-frame model, 8 for 81-frame model). Could you clarify which batch size was actually used for training?

**Comment by AsukaCamellia** [comment]
> Thank you for sharing your great work.
> 
> In the paper, you mentioned that training was conducted using two RTX 4090 GPUs and batch size of 16. Could you share how long the training process took for both the 81-frame and 243-frame models?
> 
> And, in the config files, I found that the batch size setting differs from 16 (4 for 243-frame model, 8 for 81-frame model). Could you clarify which batch size was actually used for training?


Please note that in line 52 of the train.py file, we have employed a gradient accumulation strategy.
`  accumulation_steps = 1`
The actual batch size needs to be multiplied by this step. For the case of 243 frames, this step needs to be set to 4, and for 81 frames, it should be set to 2.
